### DISCUSSION TITLE
Pasting using context menu not working in Chrome

### DISCUSSION THREAD

**Discussion by jshster** [tinymce/tinymce#9874] [General]
Using TinyMCE 7 I have enabled the context menu and added cut/copy/paste.   When you try to paste you receive the following error: 
"Your browser doesn't support direct access to the clipboard. Please use the Ctrl+X/C/V keyboard shortcuts instead.".

I realise this is a limitation of the browser but what I'm stumped on is how could this have functioned in TinyMCE 4.7 but not now.  My clients have been using this method religiously rather than keyboard shortcuts and are completely lost.  

Anyone got any idea why it would be any different between the two versions? 

Any way to fix it?

**Comment by amanraina01** [comment]
you can refer to this stack overflow answer here. But i am not fully sure, if this is the cause of this.

https://stackoverflow.com/questions/58851217/tinymce-editor-right-click-pasting-not-working

**Comment by TheSpyder** [comment]
In TinyMCE 4 you were probably using the browser's native context menu. There's a tracking ticket for updating these features to use the modern async clipboard API:
https://github.com/tinymce/tinymce/issues/7254#issuecomment-2207489979

In the meantime you could use `contextmenu: false` to remove the custom context menu entirely.
